src/app/cli/src/app.rs

```diff
@@ -582,7 +580,8 @@ pub fn register_config_in_catalog(
     let dataset_env_vars_config = config.dataset_env_vars.as_ref().unwrap();
     match dataset_env_vars_config.mode.as_ref().unwrap() {
         DatasetEnvVarsType::Static => {
-            catalog_builder.add::<kamu_datasets_services::DatasetKeyValueServiceStaticImpl>()
+            catalog_builder.add::<kamu_datasets_services::DatasetKeyValueServiceStaticImpl>();
+            catalog_builder.add::<kamu_datasets_services::DatasetEnvVarServiceStaticImpl>();
```

Comment:
I assume that a similar change needs to be done in `kamu-node` as well?

Reply:
Yes, thats, correct